.github/workflows/linux.yml

```diff
@@ -57,12 +57,12 @@ jobs:
         run: sh validate.sh -j 2 -w ghc-9.0.1 -v --lib-only -s lib-tests
       - name: Validate lib-suite
         run: sh validate.sh -j 2 -w ghc-9.0.1 -v --lib-only -s lib-suite
-  validate-8_10_4:
-    name: validate.sh ghc-8.10.4
+  validate-8_10_7:
+    name: validate.sh ghc-8.10.7
     runs-on: ubuntu-18.04
     needs: validate-8_8_4
     container:
-      image: phadej/ghc:8.10.4-bionic
+      image: phadej/ghc:8.10.7-bionic
```

Comment:
There isn't such image. And unfortunately I don't know if there will be any time soon as `hvr-ppa` is defunct (or rather the existing things work, but new are not added).

Reply:
Thanks